graphrag_sdk/test_metrics.py

```
@@ -0,0 +1,545 @@
+                verdicts = [
+                    ContextualRecallVerdict(**item) for item in data["verdicts"]
+                ]
+                return verdicts
+
+    def is_successful(self) -> bool:
+        if self.error is not None:
+            self.success = False
+        else:
+            try:
+                self.success = self.score >= self.threshold
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)